tools/esdlcmd/esdlcmd_monitor.cpp

```diff
@@ -1055,9 +1057,15 @@ class EsdlMonitorCmd : public EsdlConvertCmd
         StringBuffer escapedTemplate;
         appendEscapedEclString(escapedTemplate, diffTemplateContent);
 
+        StringBuffer definitionName;
+
+        definitionName.setf("Monitor_ActiveTemplate_%s", optMethod.str());
```

Comment:
Better use VStringBuffer here.

Reply:
Looks good. I will test it once it is availble in th enext 7.0 release